website/static/js/fangorn.js

```diff
@@ -833,6 +836,45 @@ function expandStateLoad(item) {
     reapplyTooltips();
 }
 
+/**
+ * @param tree A Treebeard _item object for the row
+ * @param nodeID Current node._id
+ * @param file window.contextVars.file object
+ */
+function setCurrentFileID(tree, nodeID, file) {
+    var tb = this;
+    if (tb.fangornFolderIndex !== undefined && tb.fangornFolderArray !== undefined && tb.fangornFolderIndex < tb.fangornFolderArray.length) {
+        for (var i = 0; i < tree.children.length; i++) {
+            var child = tree.children[i];
+            if (nodeID === child.data.nodeId && child.data.provider === file.provider && child.data.name === tb.fangornFolderArray[tb.fangornFolderIndex]) {
+                tb.fangornFolderIndex++;
+                if (child.data.kind === 'folder') {
+                    tb.updateFolder(null, child);
+                    tree = child;
+                }
+                else {
+                    tb.currentFileID = child.id;
+                }
+            }
+        }
+    }
+}
+
+/**
+ * Scroll to the Treebeard item corresponding to the given ID
+ * @param fileID id of a Treebeard _item object
+ */
+function scrollToFile(fileID) {
+    var tb = this;
+    if (fileID !== undefined) {
+        var index = tb.returnIndex(fileID);
+        var visibleIndex = tb.visibleIndexes.indexOf(index);
+        if (visibleIndex !== -1 && visibleIndex > tb.showRange.length - 2) {
+            var scrollTo = visibleIndex * tb.options.rowHeight;
+            $('#tb-tbody').scrollTop(scrollTo);
```

Comment:
This still has not been changed. Is this supposed to be `this.select('#tb-tbody').scrollTop(scrollTo)`, @caneruguz ?